examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -847,9 +847,9 @@ def main(args):
     if args.train_text_encoder:
         text_lora_attn_procs = {}
         for name, module in text_encoder.named_modules():
-            if any(x in name for x in TEXT_ENCODER_TARGET_MODULES):
+            if name.endswith(TEXT_ENCODER_ATTN_MODULE):
```

Comment:
why the change here? Is this necessary? This makes this script be a bit out of sync with `train_text_to_image_lora.py`

Reply:
Refer to https://github.com/huggingface/diffusers/pull/3437#issuecomment-1549930848. Note that this is only for the text encoder training part. For `train_text_to_image_lora.py`, we don't essentially train the text encoder anyway. So, I think it's okay.